lib/exchange/simpleswap/simpleswap_exchange_provider.dart

```diff
@@ -227,13 +227,7 @@ class SimpleSwapExchangeProvider extends ExchangeProvider {
       case CryptoCurrency.zec:
         return 'zec';
       case CryptoCurrency.bnb:
-        return currency.tag!.toLowerCase();
-      case CryptoCurrency.usdterc20:
-        return 'usdterc20';
-      case CryptoCurrency.usdttrc20:
-        return 'usdttrc20';
-      case CryptoCurrency.usdcpoly:
-        return 'usdcpoly';
```

Comment:
this would lead to interpreting the 3 of them as just `usdt` and as far as I can see there is no other way that simpleswap would know which network is this